trailblazer/mip/trending.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+""" Get MIP data for trending."""
+
+from trailblazer.mip import files
+
+
+def parse_mip_analysis(mip_config_raw: dict, qcmetrics_raw: dict, sampleinfo_raw: dict) -> dict:
+    """Parse the output analysis files from MIP for adding info
+    to trend database
+
+    Args:
+        mip_config_raw (dict): raw YAML input from MIP analysis config file
+        qcmetrics_raw (dict): raw YAML input from MIP analysis qc metric file
+        sampleinfo_raw (dict): raw YAML input from MIP analysis qc sample info file
+    Returns:
+        dict: parsed data
+    """
+
+    ### Define output dict
+    outdata = {
+        'analysis_sex': {},
+        'at_dropout': {},
+        'family': None,
+        'duplicates': {},
+        'gc_dropout': {},
+        'genome_build': None,
+        'insert_size_standard_deviation': {},
```

Comment:
And this one

Reply:
This is an important qc-metric to trend as was decided previously. Are you adding this info from some other source?